modules/component/src/let/let.directive.ts

```diff
@@ -5,140 +5,170 @@ import {
   Input,
   NgZone,
   OnDestroy,
+  OnInit,
   TemplateRef,
   ViewContainerRef,
 } from '@angular/core';
-import { NextObserver, ObservableInput, Observer, Unsubscribable } from 'rxjs';
-import { CdAware, createCdAware } from '../core/cd-aware/cd-aware_creator';
-import { createRender } from '../core/cd-aware/creator_render';
+import { Subscription } from 'rxjs';
+import { PotentialObservable } from '../core/potential-observable';
+import { createRenderScheduler } from '../core/render-scheduler';
+import { createRenderEventManager } from '../core/render-event/manager';
 
 export interface LetViewContext<T> {
-  // to enable `let` syntax we have to use $implicit (var; let v = var)
+  /**
+   * using `$implicit` to enable `let` syntax: `*ngrxLet="obs$; let o"`
+   */
   $implicit: T;
-  // to enable `as` syntax we have to assign the directives selector (var as v)
+  /**
+   * using `ngrxLet` to enable `as` syntax: `*ngrxLet="obs$ as o"`
+   */
   ngrxLet: T;
-  // set context var complete to true (var$; let e = $error)
-  $error: boolean;
-  // set context var complete to true (var$; let c = $complete)
+  /**
+   * `*ngrxLet="obs$; let e = $error"` or `*ngrxLet="obs$; $error as e"`
+   */
+  $error: any;
```

Comment:
This change is great, having access to the error makes more sense.
I do have one question, why use `any` over `unknown` here?

Reply:
I had the same dilemma. `any` seems a better option to me because casting is not possible in the template.

With `$error: unknown` and `strictTemplates: true`, the only way to access the properties of the error object in the template and avoid compiler errors would be to use `$any`:

```html
<p *ngrxLet="obs$; $error as e">
  {{ $any(e).message }}
</p>
```